### PR TITLE
refactor playerclass and default hud settings

### PR DESCRIPTION
**Pull request by Rav3n95**

### CODE REVIEW

**Review by CsokiHUN — APPROVED** [review]

**Comment by CsokiHUN** [comment]
good work! *(no breaking change)*

File: [esx]/es_extended/server/classes/player.lua [hunk 1]
```diff
@@ -1,400 +1,425 @@
...
+							x = coords.x,
+							y = coords.y,
+							z = coords.z,
+							heading = heading or 0.0
+						}
+					end
+				end
+				self:updateCoords()
+			end)
+		end,
```
Comment (wobozkyng) [inline [esx]/es_extended/server/classes/player.lua:64]: Just reminding, this loop will continue even player disconnected. I have tested it, or did I do wrong?
esx latest develop branch, server windows 6136.

**Review by bitpredator — APPROVED** [review]

File: [esx]/es_extended/server/classes/player.lua (hunk 1 above)
Comment (CsokiHUN) [reply [esx]/es_extended/server/classes/player.lua:64]: The timeout will probably run, but the update will not. You can see it here:
```lua
if DoesEntityExist(self.ped) then
```

**Comment by Gellipapa** [comment]
Hi! @Rav3n95  @wobozkyng @bitpredator @CsokiHUN Thank you for refactor and review. I merged this pr.

**Pull request merged**